presto-main/src/main/java/io/prestosql/server/protocol/QueryResultRows.java

```diff
@@ -184,6 +180,73 @@ else if (type instanceof TimestampWithTimeZoneType) {
         return Optional.of(unmodifiableList(row));
     }
 
+    private Object roundParametricDateTime(Object value, Type type)
```

Comment:
static

Reply:
it can't be static, I'd need to pass supportsParametricDateTime everywhere